python/task_manager/background_task_manager.py

```diff
@@ -18,7 +18,7 @@
 
 from .background_task import BackgroundTask
 from .worker_thread import WorkerThread
-from .results_poller import ResultsPoller
+from .results_poller import ResultsDispatcher
```

Comment:
I will rename results_poller when I am done addressing review comments. I've come not to trust GitHub to diff the files correctly when a rename happens.